riff-raff/app/controllers/forms/DeployParameterForm.scala

```diff
@@ -23,14 +25,17 @@ case class DeployParameterForm(project:String, build:String, stage:String, actio
 }
 
 object DeployParameterForm {
+  val strategyFormat: Formatter[Strategy] = enumeratum.Forms.format(Strategy)
+
   val form = Form[DeployParameterForm](
     mapping(
       "project" -> nonEmptyText,
       "build" -> nonEmptyText,
       "stage" -> text,
       "action" -> nonEmptyText,
       "selectedKeys" -> list(deploymentKey),
-      "totalKeyCount" -> optional(number)
+      "totalKeyCount" -> optional(number),
+      "updateStrategy" -> of[Strategy](strategyFormat)
```

Comment:
Ooh, this syntax is new to me! Is this the enumeratum library?

Reply:
Not enumeratum perse, but enumeratum is generating a binder for the Form. Much like path bindables, but for forms :)